fence/agents/vbox/fence_vbox.py

```diff
@@ -21,66 +22,82 @@
 BUILD_DATE = ""
 #END_VERSION_GENERATION
 
-
-def get_name_or_uuid(options):
-    return options.get("--uuid") or options.get("--plug")
-
-_domain_re = re.compile(r'^\"(.*)\" \{(.*)\}$')
-
-
 def _invoke(conn, options, *cmd):
     prefix = options["--sudo-path"] + " " if "--use-sudo" in options else ""
-    conn.sendline(prefix + "VBoxManage " + " ".join(cmd))
+    conn.send_eol(prefix + options["--vboxmanage-path"] + " " + " ".join(cmd))
+    conn.log_expect(options["--command-prompt"], int(options["--shell-timeout"]))
```

Comment:
Aah, I just wanted to reuse branch. fence_vbox should not be part of this commit.